packages/flutter/test/widgets/draggable_test.dart

```diff
@@ -265,7 +265,7 @@ void main() {
             },
             onMove: (DragTargetDetails<int> details) {
               targetMoveCount['Target 1'] =
-                  targetMoveCount['Target 1']! + details.data;
+                  targetMoveCount['Target 1']! + details.data!;
```

Comment:
Hey @Piinks, this is where the problem would arise in old codebases.

Reply:
I think there would be more than one problem, running tests though to find out